resources/tentacle.rb

```diff
@@ -29,6 +29,11 @@
 attribute :app_path, kind_of: String, default: 'C:\Octopus\Applications'
 attribute :trusted_cert, kind_of: String
 attribute :port, kind_of: Fixnum, default: 10_933
+attribute :polling_port, kind_of: Fixnum, default: 10_943
```

Comment:
I think we should just use the same port value for everything.  If its so we can have an intelligent default we can lazy evaluate the default and make it look at `polling` or not